pkg/storaged/dialog.jsx

```diff
@@ -724,100 +725,50 @@ export const SelectSpaces = (tag, title, options) => {
-                        const desc = block === spc.desc ? "" : spc.desc;
-                        const on_change = (event) => {
-                            if (event.target.checked)
-                                change(spc);
-                        };
-
-                        return (
-                            <DataListItem key={spc.block ? spc.block.Device : spc.desc}>
-                                <DataListItemRow>
-                                    <div className="pf-v5-c-data-list__item-control">
-                                        <div className="pf-v5-c-data-list__check">
-                                            <input type='radio' value={desc} name='space' checked={val == spc} onChange={on_change} />
-                                        </div>
-                                    </div>
-                                    <DataListItemCells
-                                        dataListCells={[
-                                            <DataListCell key="select-space-name" className="select-space-name">
-                                                {format_size_and_text(spc.size, desc)}
-                                            </DataListCell>,
-                                            <DataListCell alignRight isFilled={false} key="select-space-details" className="select-space-details">
-                                                {block}
-                                            </DataListCell>,
-                                        ]}
-                                    />
-                                </DataListItemRow>
-                            </DataListItem>
-                        );
-                    })
-                    }
-                </DataList>
+                            return (
+                                <Tr key={spc.block ? spc.block.Device : spc.desc}
+                                    onRowClick={() => on_change(null, !selected)}>
+                                    <Td select={{
+                                        // rowIndex is important for state management,
+                                        // without it, onRowClick has no visible effect
+                                        rowIndex: row_index,
+                                        onSelect: on_change,
+                                        isSelected: selected,
+                                    }} />
+                                    <Td>{block}</Td>
+                                    <Td>{spc.name}</Td>
+                                    <Td>{spc.subname}</Td>
+                                    <Td>{spc.type_label}</Td>
+                                    <Td modifier="nowrap">{fmt_size(spc.size)}</Td>
+                                </Tr>
+                            );
+                        })
+                        }
+                    </Tbody>
+                </Table>
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18967-20230629-132526-4d4c19dd-fedora-38-devel-other//Coverage/lcov/github-pr.diff.gcov.html)